recipe/meta.yaml

```diff
@@ -150,15 +180,13 @@ test:
 
     # run tests except tests_to_skip
     {% set param = "verbose=1, label=" + label + ", tests=None" %} # set verbose to 2 for additional logs
-    {% set extra = "extra_argv=['-k', 'not (" + tests_to_skip + " or " + tests_known_to_crash + ")', '-n', '3', '--timeout=1800', '--durations=50']" %}
+    {% set extra = "extra_argv=['-k', 'not (" + tests_to_skip + ")', '-n', '3', '--timeout=1800', '--durations=50']" %}
     - python -c "import scipy, sys; sys.exit(not scipy.test({{ param }}, {{ extra }}))"
 
     # run tests_to_skip tests, but ignore results (to keep a record of failures)
     {% set param_fail = "verbose=2, label=" + label + ", tests=None" %}
-    {% set extra_crash = "extra_argv=['-k', '(" + tests_known_to_crash + ")', '-n', '3', '--timeout=1800', '--durations=50']" %}
     {% set extra_fail = "extra_argv=['-k', '(" + tests_to_skip + ")', '-n', '3', '--timeout=1800', '--durations=50']" %}
-    - python -c "import scipy, sys; scipy.test({{ param_fail }}, {{ extra_crash }}); sys.exit(0)" # [not s390x]
-    - python -c "import scipy, sys; scipy.test({{ param_fail }}, {{ extra_fail }}); sys.exit(0)"  # [not s390x]
+    - python -c "import scipy, sys; scipy.test({{ param_fail }}, {{ extra_fail }}); sys.exit(0)"
 
 about:
   home: https://scipy.org/
```

Comment:
Could you please update `license` and `license_file`?

`grayskull` generated this (+ I added `LICENSES_bundled.txt`):
```
  license: BSD-3-Clause AND MIT AND BSD-3-Clause-Attribution AND BSD-2-Clause
  license_file:
    - LICENSE.txt
    - LICENSES_bundled.txt
    - scipy/ndimage/LICENSE.txt
    - scipy/optimize/tnc/LICENSE
    - scipy/optimize/_direct/COPYING
    - scipy/io/_fast_matrix_market/LICENSE.txt
    - scipy/io/_fast_matrix_market/fast_matrix_market/dependencies/fast_float/LICENSE-MIT
    - scipy/io/_fast_matrix_market/fast_matrix_market/dependencies/ryu/LICENSE-Boost
    - scipy/_lib/boost_math/LICENSE
    - scipy/_lib/_uarray/LICENSE
    - scipy/_lib/array_api_compat/LICENSE
    - scipy/_lib/pocketfft/LICENSE.md
    - scipy/_lib/unuran/license.txt
    - scipy/_lib/highs/LICENSE
    - scipy/_lib/highs/extern/pdqsort/license.txt
    - scipy/_lib/highs/extern/filereaderlp/LICENSE
    - scipy/fft/_pocketfft/LICENSE.md
    - scipy/sparse/linalg/_eigen/arpack/ARPACK/COPYING
    - scipy/sparse/linalg/_propack/PROPACK/license.txt
    - scipy/sparse/linalg/_dsolve/SuperLU/License.txt
    - scipy/spatial/qhull_src/COPYING.txt
    - scipy/stats/biasedurn/license.txt
```

See also https://github.com/scipy/scipy/blob/v1.13.0/LICENSES_bundled.txt

Reply:
updated